capstone-app/src/app/modules/history/wiki-media/wiki-media.component.html

```diff
@@ -4,9 +4,12 @@
       <mat-card-title class="history-title">
         {{title}}
       </mat-card-title>
-      <div class="text">
-      {{body}}
+      <div class="text" *ngFor="let paragraph of body">
+        <br/>
+        <p>{{paragraph}}</p>
       </div>
+      <br/>
+      <i>All the data above was obtained<a href="{{UrlForAttribution}}" target="_blank"> here.</a></i>
```

Comment:
Add a space after "obtained" and remove space before "here", so that the space isn't part of the link. Also, move the period after the end of the anchor tag.